examples_gbs/run_tutorial_points.py

```diff
@@ -0,0 +1,137 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports,invalid-name
+r"""
+Point Process Tutorial
+======================
+
+This tutorial shows how to generate GBS point process samples and use them to detect outlier
```

Comment:
I would cite your paper more prominently. Perhaps also have a line that makes clear that the `points` module has most of the core functionality for this tutorial